frontend/public/module/k8s/pods.ts

```diff
@@ -234,7 +234,7 @@ export const podPhase = (pod: PodKind): PodPhase => {
   let phase = pod.status.phase || pod.status.reason;
 
   _.each(pod.status.initContainerStatuses, (container: ContainerStatus, i: number) => {
-    const { terminated, waiting } = container.state;
+    const { terminated, waiting, running, ready } = container.state;
     if (terminated && terminated.exitCode === 0) {
       return true;
     }
```

Comment:
I think the [k8s printPod func](https://github.com/kubernetes/kubernetes/blob/c75e83302660750dc61f1b4a967ca52fc767df3b/pkg/printers/internalversion/printers.go#L915-L920) handles sidecars here and continues the loop.

Reply:
> k8s printPod func

Oh something like this, right?
`    if ((running && ready)) {
      return true;
    }`